amd_openvx/openvx/ago/ago_haf_cpu_geometric.cpp

```diff
@@ -1187,8 +1187,8 @@ vx_uint8			* pLocalData
 				mask.f = _mm_and_ps(mask.f, _mm_cmplt_ps(xmap, srcbx));
 				mask.f = _mm_and_ps(mask.f, _mm_cmpge_ps(ymap, zero));
 				mask.f = _mm_and_ps(mask.f, _mm_cmplt_ps(ymap, srcby));
-				int m = _mm_movemask_ps(mask.f);
-				if (m){
+				//int m = _mm_movemask_ps(mask.f);
```

Comment:
remove commented lines

Reply:
I dont think we have touched this file in HIP. Maybe the tab to space conversion and other changes for cpu files can be done in a different PR